src/main/Subsystems/Storage/Storage.cpp

```diff
@@ -14,19 +14,23 @@ void Storage::setVoltage(units::volt_t voltage) {
 
 bool Storage::isNoteOnForwardSensor() {
 	// return !forwardSensor.Get();
-	return noteOnForwardCache;
+	//return noteOnForwardCache;
+	return IRvalue > 20;
```

Comment:
Add the threshold for note detection to constant file

Reply:
Fixed